Sources/App/Logger/LogzIOLogger.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+import Vapor
+
+public final class LogzIOLogger: Logger, Service {
```

Comment:
Can we call this `JSONLogger` instead? Logz is just an implementation/service detail and the structure is very generic.

Reply:
Fixed in e42eaa2